extensions/amp-ad/amp-ad.md

```diff
@@ -182,33 +184,38 @@ This is due to the UX implications of full page overlay ads. It may be considere
 
 ## Attributes
 
-### type
+**type**
```

Comment:
I think attributes shouldn't be titles - they render awfully.  Tables would be better, which I might do in the next round of updates - after the agency changes the site.  Actually, in most dev docs, attributes are description list (`<dl>` `<dt>` .. etc.) which looks better but requires a lot of html.  I'm still pondering it, but for now, I felt using bold was the best option.